agent/security_helper.py

```diff
@@ -62,6 +74,7 @@ def netstat_scan():
             'port': c.laddr[1],
             'proto': {SocketKind.SOCK_STREAM: 'tcp', SocketKind.SOCK_DGRAM: 'udp'}.get(c.type),
             'state': c.status if c.type == socket.SOCK_STREAM else None,
+            'process_info': get_process_info(c.pid)
```

Comment:
we already send the full list of processes to the server, why this extra field?

Reply:
> we already send the full list of processes to the server, why this extra field?

fixed